Desktop/html/js/jaspwidgets.js

```diff
@@ -1009,6 +1010,97 @@ JASPWidgets.Toolbar = JASPWidgets.View.extend({
 	}
 })
 
+JASPWidgets.RSyntaxModel = Backbone.Model.extend({
+	defaults: {
+		analysis: {},
+		script: ""
+	},
+	getFromAnalysis: function(item) {
+		return this.attributes.analysis.model.get(item);
+	}
+
+});
+
+JASPWidgets.RSyntaxView = JASPWidgets.View.extend({
+	initialize: function() {
+		this.$el.addClass("jasp-rsyntax-container");
+		this.$el.addClass("jasp-hide");
+		this.$el.addClass("jasp-code");
+		this._insertRSyntax()
+	},
+	setVisibility: function (value) {
+		var self = this;
+		self.$el.css("opacity", value ? 0 : 1);
+
+		if (value === true) {
+			self.$el.slideDown(200, function () {
+				self._setVisibility(value);
+				self.$el.animate({ "opacity": 1 }, 200, "easeOutCubic", function () {
+					window.scrollIntoView(self.$el, function () {});
```

Comment:
If you remove window.scrollIntoView(self.$el, function () {});` the behaviour is correct again as far as I can see